### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --hidden --format brief`

### CODE REVIEW

File: tests/analysers/hf_checker_test.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def analyser():
     (
         BaseDocument(
             'DUMMY_URL',
-            'This is a long enough sentence with an eror or tvo.',
```
Comment (hangyav) [inline tests/analysers/hf_checker_test.py:31]: This is an intentional error. Please revert.

File: tests/analysers/hf_instruction_checker_test.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def analyser():
     (
         BaseDocument(
             'DUMMY_URL',
-            'This is a long enough sentence with an eror or tvo.',
```
Comment (hangyav) [inline tests/analysers/hf_instruction_checker_test.py:30]: This is an intentional error. Please revert.

File: tests/documents/document_test.py [hunk 3]
```diff
@@ -542,7 +542,7 @@ def test_updates(content, edits, exp):
         ("too short to detect", "auto", BaseDocument.DEFAULT_NATURAL_LANGUAGE),
         ("too short to detect", None, BaseDocument.DEFAULT_NATURAL_LANGUAGE),
         ("This is a document. It has more than twenty characters in it.", "auto", "en"),
-        ("Das ist ein Dokument. Es enthält mehr als zwanzig Zeichen.", "auto", "de"),
```
Comment (hangyav) [inline tests/documents/document_test.py:545]: "als" is a correct German word. Please revert.

**Review by hangyav — CHANGES_REQUESTED** [review]
There are some changes to be reverted.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**